biogrid/apps/biogrid-mvp/src/app/components/simulation-board.tsx

```diff
@@ -0,0 +1,129 @@
+import './simulate-board.css';
+
+export interface GridItemRet {
+  gridItemName: string;
+  relativePosition: ItemPosition;
+}
+
+export interface GridItemLines {
+  fromItem: string;
+  toItem: string;
+}
+
+interface SimBoardProps {
+  height: Distance;
+  width: Distance;
+  items: GridItemRet[];
+  lines: GridItemLines[];
+}
+
+interface ItemsByName {
+  [name: string]: ItemPosition;
+}
+
+const Grid = () => {
+  return (
+    <div className="grid-item">
+      <GridSvg />
+    </div>
+  );
+};
+const LargeBattery = () => {
+  return (
+    <div className="grid-item">
+      <LargeBatterySvg />
+    </div>
+  );
+};
+const SmallBattery = () => {
+  return (
+    <div className="grid-item">
+      <SmallBatterySvg />
+    </div>
+  );
+};
+const House = () => {
+  return (
+    <div className="grid-item">
+      <HouseSvg />
+    </div>
+  );
+};
+const SolarPanel = () => {
+  return (
+    <div className="grid-item">
+      <SolarPanelSvg />
+    </div>
+  );
+};
+
+export const SimulationBoard = (props: SimBoardProps) => {
+  const itemsByName: ItemsByName = props.items.reduce(
+    (map: ItemsByName, item) => {
+      map[item.gridItemName] = item.relativePosition;
+      return map;
+    },
+    {}
+  );
+  const kilometersToCSSWidth = (distance: Distance) => {
+    return `${Math.ceil((distance / props.width) * 100)}%`;
+  };
+  const kilometersToCSSHeight = (distance: Distance) => {
+    return `${Math.ceil((distance / props.height) * 100)}%`;
+  };
+  const boardWidth = '50vw';
+  const boardHeight = `${Math.ceil((props.height / props.width) * 50)}vw`;
+  return (
+    <div
+      className="simulation-board"
+      style={{
+        width: boardWidth,
+        height: boardHeight,
+      }}
+    >
+      {props.items.map((item, i) => (
+        <div
+          className={`grid-item-wrapper ${item.gridItemName}`}
+          style={{
+            top: kilometersToCSSHeight(item.relativePosition.y),
+            left: kilometersToCSSWidth(item.relativePosition.x),
+          }}
+        >
+          {item.gridItemName.includes('small_battery') ? (
+            <SmallBattery />
+          ) : item.gridItemName.includes('large_battery') ? (
+            <LargeBattery />
+          ) : item.gridItemName.includes('solar_panel') ? (
+            <SolarPanel />
+          ) : item.gridItemName.includes('grid') ? (
+            <Grid />
+          ) : (
+            <House />
+          )}
+        </div>
+      ))}
+      <div className="grid-line-wrapper">
+        <svg width={boardWidth} height={boardHeight}>
+          {props.lines.map((line) => (
+            <line
+              x1={`${kilometersToCSSWidth(itemsByName[line.fromItem].x)}`}
+              y1={`${kilometersToCSSHeight(itemsByName[line.fromItem].y)}`}
+              x2={`${kilometersToCSSWidth(itemsByName[line.toItem].x)}`}
+              y2={`${kilometersToCSSHeight(itemsByName[line.toItem].y)}`}
```

Comment:
nit: looks like these are going from grid corner to grid corner, might help the UI if these were go from center of side edge to center of side edge

Reply:
fixed!